subsys/bluetooth/controller/ll_sw/ctrl.c

```diff
@@ -2277,55 +2277,60 @@ isr_rx_conn_pkt_ctrl_rej_phy_upd(struct radio_pdu_node_rx *node_rx,
 static inline void
 isr_rx_conn_pkt_ctrl_rej(struct radio_pdu_node_rx *node_rx, u8_t *rx_enqueue)
```

Comment:
@cvinayak There is similar code in the SPLIT controller. But I think this bug is a product of the parallel procedure handling. So I'm not certain if the fix should be applied there yet.